docs/api/index.html

```diff
@@ -609,7 +609,7 @@
 <pre><code><span class="xml">  <span class="hljs-tag">&lt;<span class="hljs-name">Griddle</span> <span class="hljs-attr">data</span>=</span></span><span class="hljs-template-variable">{data}</span><span class="xml"><span class="hljs-tag">&gt;</span>
     <span class="hljs-tag">&lt;<span class="hljs-name">RowDefinition</span>&gt;</span>
       <span class="hljs-tag">&lt;<span class="hljs-name">ColumnDefinition</span> <span class="hljs-attr">id</span>=<span class="hljs-string">"name"</span> <span class="hljs-attr">title</span>=<span class="hljs-string">"Name"</span> /&gt;</span>
-      <span class="hljs-tag">&lt;<span class="hljs-name">ColumnDefinition</span> <span class="hljs-attr">id</span>=<span class="hljs-string">"company"</span> <span class="hljs-attr">title</span>=<span class="hljs-string">"Company order=</span></span></span><span class="hljs-template-variable">{1}</span><span class="xml"><span class="hljs-tag"><span class="hljs-string"> /&gt;
+      <span class="hljs-tag">&lt;<span class="hljs-name">ColumnDefinition</span> <span class="hljs-attr">id</span>=<span class="hljs-string">"company"</span> <span class="hljs-attr">title</span>=<span class="hljs-string">"Company" order=</span></span></span><span class="hljs-template-variable">{1}</span><span class="xml"><span class="hljs-tag"><span class="hljs-string"> /&gt;
```

Comment:
There should be a `</span>` after the new `"` to close out the `hljs-string` followed by a new `<span>` to start the `hljs-attr`.